cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -208,6 +210,13 @@ var createModeSubcommands = map[string]modeSubcommand{
 				"MinReplicas denotes the minimum number of desired Pods for this worker group.")
 			subcmd.Flags().StringToIntVar(&o.MaxReplicas, maxReplicasFlagName, nil,
 				"MaxReplicas denotes the maximum number of desired Pods for this worker group, and the default value is maxInt32.")
+			subcmd.Flags().StringVar(&o.RayCluster, rayClusterFlagName, "",
+				"Ray cluster name where ray job will crate.")
```

Comment:
can we reformulate this? 

"Existing cluster on which the job will be created" or something.

Reply:
Fixed. Thanks.